code/src/components/Form.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+
+import { UserName } from './Name'
+import { AgeRadioOption } from './Age'
+import { SelectPlanet } from './SelectPlanet'
+import { BringItem } from './BringItem'
+import { Summary } from './Summery'
+
+
+
+
+export const Form = () => {
+  const [ageGroupSpan, setAgeGroup] = useState('')
+  const [name, setName] = useState('')
+  const [item, setInputItem] = useState('')
```

Comment:
Same comment really as in line 13. My suggestion is to either change name to “inputItem” or set function to “setItem”. 🙂